.github/workflows/new-project.yml

```diff
@@ -0,0 +1,75 @@
+name: Create new project and run tests etc.
+
+on:
+  push:
+    branches:
+    - dev_refactor
+  pull_request:
+    branches:
+    - dev_refactor
+
+jobs:
+  new-project:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9.7
+    - uses: conda-incubator/setup-miniconda@v2
+      with:
+        miniconda-version: "latest"
+        channels: conda-forge
+        channel-priority: flexible
+        show-channel-urls: true
+    - name: Create blueprint run env from unpinned reqs
+      run: |
+        conda env create --name blueprint --file requirements/requirements.yml
+    - name: Create new project
+      run: |
+        conda run --name blueprint cookiecutter --no-input .
+    - name: Prepare new project
+      working-directory: ./flying_circus
+      run: |
+        git init
+        git config user.name "Monty Python"
+        git config user.email "monty.python@meteoswiss.ch"
+        git add .
+        git commit -m "initial commit"
```

Comment:
To make sure that creating a new project actually works. Because of the jinj2 templates (`{{ project_slug }}` et al.), linters etc. are not applicable to the sample code etc. w/o creating a new project. Checking the rest of the blueprint doesn't really ensure that users start with a project that actually works.

Whether github actions is the right place for that (instead of Jenkins) I'm not sure, or in what instances/how often this should be run. But unless there are hard resource constraints, I'd keep it.